node/tests/SharedTests.ts

```diff
@@ -18,6 +18,17 @@ import {
     Script,
     parseInfoResponse,
 } from "../";
+import { SingleNodeRoute } from "../build-ts/src/GlideClusterClient";
+import {
+    BitOffsetOptions,
+    BitmapIndexType,
+} from "../build-ts/src/commands/BitOffsetOptions";
+import { ConditionalChange } from "../build-ts/src/commands/ConditionalChange";
```

Comment:
Don't move imports please, just add new ones

Reply:
i think that happened when I was rebasing my work earlier. Should I rebase again now?